ISSUETEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+# LGT GridIron Project
+
+### Issue Title:
+[Concisely summarize the issue or feature request in a few words]
+
+### Description:
+[Provide a detailed description of the issue or feature request. Include any relevant context or background information that might be helpful for understanding the problem.]
```

Comment:
W: I'm honestly torn between keeping these sentences separate and combining them into one unified bracket like the following:
[Provide a detailed description of the issue or feature request. Include any relevant context or background information that might be helpful for understanding the problem. Any additional information or context that may be relevant to the issue. This could include workaround solutions, related issues, etc.]